.github/workflows/testall.yml

```diff
@@ -1,27 +1,32 @@
-name: test workflow
+name: Run tests
 
 on:
   push
 
 jobs:
-  build:
+  runtests:
+    name: Run cppcheck, valgrind and diff
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Install
+      - name: Install dependencies for valgrind
         run: sudo apt-get install -y valgrind cppcheck
-      - name: Cppcheck warning
-        run: cppcheck *.cpp --enable=warning --error-exitcode=1
-      - name: Upload CPPCheck report
-        run: cppcheck *.cpp --enable=all --output-file=cppreport.txt
-      - uses: actions/upload-artifact@v2
+      - name: run Makefile
+        run: make all
```

Comment:
itt modnjuk minek csinalod meg a doxy-t is pl?

Reply:
`all: runGame cppcheck cppcheckup valgrind diff`
make all paranccsal csak a Makefile 17. sorában jelölt targetek futnak le, elég logikátlan módon jelöltem, ezt azt **all**t átneveztem **tests**-re